src/main/scala/codacy/scalastyle/ScalaStyle.scala

```diff
@@ -0,0 +1,65 @@
+package codacy.scalastyle
+
+import java.net.URLClassLoader
+import java.nio.file.Paths
+
+import better.files.File
+import codacy.docker.api.metrics.{FileMetrics, MetricsTool}
+import codacy.docker.api.{MetricsConfiguration, Source}
+import com.codacy.api.dtos.Language
+import org.scalastyle._
+
+import scala.util.Try
+
+object ScalaStyle extends MetricsTool {
+
+  override def apply(source: Source.Directory,
+                     language: Option[Language],
+                     files: Option[Set[Source.File]],
+                     options: Map[MetricsConfiguration.Key, MetricsConfiguration.Value]): Try[List[FileMetrics]] = {
+
+    val filesToLint = files.map(_.map(file => source.path + "/" + file.path)).getOrElse(Set(source.path))
+
+    val config = MainConfig(error = false, directories = filesToLint.toList)
```

Comment:
`directories` here seems weird since we are passing files.

Reply:
This comes from the jar. I didn't name it.